KubeArmor/BPF/system_monitor.c

```diff
@@ -782,6 +779,23 @@ int kprobe__security_path_unlink(struct pt_regs *ctx){
 
     return 0;
 }
+
+#if defined(SECURE_PATH_UNLINK)
+SEC("kprobe/security_path_unlink")
+int kprobe__security_path_unlink(struct pt_regs *ctx){
+    if (skip_syscall())
+		return 0;
+    return security_path__dir_path_args(ctx);
+}
+#endif
+
+#if defined(SECURE_PATH_RMDIR)
```

Comment:
Since they belong to the same family of hooks 
https://elixir.bootlin.com/linux/latest/source/include/linux/security.h#L1769
Should we be using different flags to check and load them?

Reply:
done